jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/DatePickersTests.java

```diff
@@ -351,13 +348,9 @@ public void expandableDatePicker() {
         pickerInMenuDatePicker.changeYear();
         pickerInMenuDatePicker.selectYear(Integer.toString(currentYear - 99));
         pickerInMenuDatePicker.has().year(Integer.toString(currentYear - 99));
-        buttonOkMenu.clickOk();
-        pickerInMenuDatePicker.expand();
         pickerInMenuDatePicker.has().mainDateFieldIsNotExist();
-        buttonCancelMenu.clickCancel();
         pickerInDialogDatePicker.expand();
         pickerInDialogDatePicker.has().mainDateField();
-        buttonOkDialog.clickOk();
     }
```

Comment:
Just to clarify, after changes we do not check functionality of the buttons 'OK' and 'CANCEL'?

Reply:
Yes, because we don't have access to them after changes.